folder-from/file-to-move.md

```diff
@@ -0,0 +1 @@
+Line to add a comment to
```

Comment:
This is a comment that should become outdated once the file is moved to another folder.